ecs-init/config/awsrulesfn/partition.go

```diff
@@ -0,0 +1,76 @@
+package awsrulesfn
```

Comment:
[This](https://github.com/aws/amazon-ecs-agent/blob/85a28b923aeb9a6a61e818c8cf4d34892fd49f83/ecs-init/config/awsrulesfn/partitions.gen.go) `go generate` copies [this AWS SDK file](https://github.com/aws/aws-sdk-go-v2/blob/main/internal/endpoints/awsrulesfn/partition.go) from vendor into our package.